Examples/Hyperspace-iOSExample/ViewController.swift

```diff
@@ -19,10 +19,16 @@ class ViewController: UIViewController {
     // MARK: - Properties
     
     private let backendService = BackendService(networkService: NetworkService(networkActivityIndicatable: UIApplication.shared))
-    private var trustValidatingBackendService: BackendService?
+    private lazy var trustValidatingBackendService: BackendService = {
+        // Create's a trust configuration whereby requests to the domain 'jsonplaceholder.typicode.com' will be validated to ensure they are being served a certificate matchin 'jsonplaceholder.der'
```

Comment:
Small typo in comment: "matchin" --> "match in".

Reply:
Typo fixed.